api/src/endpoints/index.js

```diff
@@ -1,7 +1,6 @@
 import express from 'express';
 import neo4jRoutes from 'endpoints/neo4j';
 import repoRoutes from 'endpoints/repository';
-import hpaRoutes from 'endpoints/hpaRna';
```

Comment:
This looks like a safe removal but not relevant to the linked issue. I specifically wanted to mention it here because we have associated code in the `frontend` that can also be removed.

Reply:
To my understanding, this removal was meant to be part of the epic https://github.com/MetabolicAtlas/private-issues/issues/124, specifically issue https://github.com/MetabolicAtlas/MetabolicAtlas/pull/699 or https://github.com/MetabolicAtlas/data-generation/pull/23 so this is meant just to fix the message shown in `logs api`.

Reply:
It's true that we no longer need it but I wanted to mention it since the frontend has some files that should be removed too along with this, for example: https://github.com/MetabolicAtlas/MetabolicAtlas/blob/develop/frontend/src/store/modules/humanProteinAtlas.js

Reply:
I agree with @shan-chalmers that the removal of this file is irrelevant to this issue. But we can keep the change this time.